safep/fileIO.py

```diff
@@ -128,4 +128,136 @@ def read_Files(files, step=1):
     data.index = data.window
     data["dVdW"] = data.vdW_ldl - data.vdW_l
     
-    return data
\ No newline at end of file
+    return data
+
+
+def parse_Colvars_log(filename):
```

Comment:
This function will need to be refactored for length, either now or later.

Reply:
If will be refactored by moving out of SAFEP :-)

Reply:
What is your criterion for function length?